### PR TITLE
Added support for displaying Maven logs in JSON

### PR DESCRIPTION
**Pull request by orpiske**
This solves GH issue #1783

**Release Note**
```release-note
Added support for displaying Maven logs in JSON format
```

### CODE REVIEW

**Comment by orpiske** [comment]
The Operator container build is broken on some scenarios. I am looking at that.

**Review by oscerd — APPROVED** [review]

File: .github/workflows/builder.yml [hunk 1]
```diff
@@ -85,6 +85,8 @@ jobs:
       run: |
         echo "Build project"
         make build-kamel bundle-kamelets
+        echo "Preparing the maven overlay"
```
Comment (astefanutti) [inline .github/workflows/builder.yml:88]: I wonder whether we could call `make images` to avoid duplicating the build logic across workflows?

Comment (orpiske) [reply .github/workflows/builder.yml:88]: Yeah. I think there's some duplication on the actions code and we could probably clean it. My .2 cents: let's open a ticket for this and I'll take care of it on a separate PR. wdyt?

Comment (astefanutti) [reply .github/workflows/builder.yml:88]: Sounds good to me 👍🏼.

**Review by astefanutti — APPROVED** [review]

File: .github/workflows/builder.yml (hunk 1 above)
Comment (orpiske) [reply .github/workflows/builder.yml:88]: Deal. I'll take care of the ticket and the PR for this.

**Pull request merged**